docs/codespace_pr_reviews.md

```diff
@@ -0,0 +1,29 @@
+# Using Codesapces to review PRs without pulling a repo to your machine
+
+1. go to the PR
+2. click the `Code` button in the top right corner
+3. click the `Codespaces` tab
+4. `Create new codespace`
+
+   ![Create a new codespace screenshot](https://prod-files-secure.s3.us-west-2.amazonaws.com/bf5cb10c-8e7c-41db-97ca-ded0d1f001c9/49ab6248-8f67-4e79-9ee5-7dee09e03237/Untitled.png)
```

Comment:
Odd, i'm not seeing the exact language from your screenshot 
<img width="385" alt="image" src="https://github.com/trussworks/react-uswds/assets/764090/54bd678c-77e5-46d0-b008-bbb41ca61de4">

Reply:
in my screenshot, I had had a codespace on the branch previously, i didn't realize that would make a difference. do you think it's confusing?

Reply:
same, i created then deleted. Probably fine and followable and likely to change again anyway

Reply:
i see what you mean now, I've updated it to reflect "create a new codespace on" language to make it less confusing.